README.md

```diff
@@ -57,151 +52,251 @@ To save your changes, enter the following in your command line:
     source ~/.bash_profile
 
 
-## Usage
+# Usage
 
-The FareHarbor External API Gem is a wrapper for the FareHarbor External API.  Each endpoint returns data only for companies that the affiliate / API-user has permission to access.
+The FareHarbor External API Gem is a wrapper for the FareHarbor External API.  Each endpoint of the API returns data for companies associated with an affiliate or that the API-user has permission to access.
```

Comment:
I think maybe this line should be moved up as an introductory sentence for the gem?